src/components/Confirm/style.css

```diff
@@ -1,14 +1,41 @@
 @import (less) "../Theme/constants.css";
 
-.title {
-  padding-bottom: 0px;
+.previewsContainer {
+  /*
+  The height of this div needs to be calculated minus the margin top and bottom of its parent
+  This will allow the div to have a consistent height and spacing between its children
+  It will also allow the actions to always sit above the "powered-by-onfido" logo
+   */
+  height: calc(~"100% - 112px");
+  width: 100%;
+  position: absolute;
+  display: flex;
```

Comment:
how is flex being used?

Reply:
I am using flex to evenly display the elements within the screen. In this case I want to display them vertically and space them evenly in this order:
- Title
- Capture
- Actions
This will allow the margins between each div to change depending on the content, ie when we have multiline content. 
To see how this is working check out the confirm screen and see how it changes when the locale changes.

Reply:
cool, thanks for the info!